search/api.py

```diff
@@ -228,6 +235,27 @@ def create_user_mapping():
             'state_or_territory': NOT_ANALYZED_STRING_TYPE,
         }},
     })
+    mapping.field('enrollments', 'nested', properties={
+        'course_details': {
```

Comment:
I think what George did was extracting the actual enrollment information from the nested part of the JSON: I actually did not pay much attention to this specific part.